### PR TITLE
Create wisiwii.md

### PR DESCRIPTION
**Pull request by wisiwii**
Let's comment 👾

### CODE REVIEW

File: docs/_slides/wisiwii.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from [username]!
+
+![dinotocat](https://octodex.github.com/privateinvestocat/)
```
Comment (wisiwii) [inline docs/_slides/wisiwii.md:8]: what do you think @unaizorrilla ?

Comment (unaizorrilla) [reply docs/_slides/wisiwii.md:8]: Look godd but i prefer

```suggestion
![dinotocat](https://octodex.github.com/images/luchadortocat.png)
```

File: docs/_slides/wisiwii.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
```
Comment (unaizorrilla) [inline docs/_slides/wisiwii.md:3]: take care on case

File: docs/_slides/wisiwii.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+---
+
+Greetings from [username]!
```
Comment (unaizorrilla) [inline docs/_slides/wisiwii.md:6]: please, set your nickname instead of the [username] bookmark

**Review by unaizorrilla — CHANGES_REQUESTED** [review]
LGTM but fix the [username] not replaced

File: docs/_slides/wisiwii.md (hunk 3 above)
Comment (unaizorrilla) [inline docs/_slides/wisiwii.md:6]:
```suggestion
Greetings from wisiwii!
```

File: docs/_slides/wisiwii.md (hunk 2 above)
Comment (wisiwii) [reply docs/_slides/wisiwii.md:3]: I see it ok @unaizorrilla ?

**Review by unaizorrilla — APPROVED** [review]

**Review by unaizorrilla — APPROVED** [review]

**Pull request merged**